README.md

```diff
@@ -1 +1,3 @@
-# M2-Embedded_Seat-Heat-Control-
\ No newline at end of file
+# M2-Embedded_Seat-Heat-Control-
+
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/68a4e8525d8d4edf81e81f254e874313)](https://app.codacy.com/gh/PrasadVallem/M2-Embedded_Seat-Heat-Control?utm_source=github.com&utm_medium=referral&utm_content=PrasadVallem/M2-Embedded_Seat-Heat-Control&utm_campaign=Badge_Grade_Settings)
```

Comment:
Codacy has a fix for the issue: [Files should end with a single newline character](https://app.codacy.com/gh/PrasadVallem/M2-Embedded_Seat-Heat-Control/pullRequest?prid=8498617)
```suggestion
[![Codacy Badge](https://api.codacy.com/project/badge/Grade/68a4e8525d8d4edf81e81f254e874313)](https://app.codacy.com/gh/PrasadVallem/M2-Embedded_Seat-Heat-Control?utm_source=github.com&utm_medium=referral&utm_content=PrasadVallem/M2-Embedded_Seat-Heat-Control&utm_campaign=Badge_Grade_Settings)

```